tpmutil/run.go

```diff
@@ -25,37 +25,48 @@ import (
 	"io"
 	"net"
 	"os"
+	"strings"
 )
 
-// OpenTPM opens a channel to the TPM at the given path. If the file is a
-// device, then it treats it like a normal TPM device, and if the file is a
-// Unix domain socket, then it opens a connection to the socket.
+const localSocketPrefix = "localhost:"
+
+// OpenTPM opens a channel to the TPM at the given path.
+//
+// If path is a local TCP socket (i.e. starts with "localhost:"), it's dialed
+// and returned.
+// If path is a unix socket, it's dialed and returned.
+// If path is a device file, then it's opened and returned.
 func OpenTPM(path string) (io.ReadWriteCloser, error) {
-	// If it's a regular file, then open it
-	var rwc io.ReadWriteCloser
+	// If it's a local TCP socket, dial it.
+	if strings.HasPrefix(path, localSocketPrefix) {
+		con, err := net.Dial("tcp", path)
```

Comment:
Go is all for short names, so maybe this could be `c`. But otherwise, I'd like to use `conn` for this (two n's), since that's the spelling used, e.g., in net.TCPConn. Or see the first example in the net package (https://golang.org/pkg/net/)